contents/docs/getting-started/group-analytics.mdx

```diff
@@ -1,70 +1,9 @@
 ---
-title: Group analytics
+title: Groups
 nextPage: ./next-steps.mdx
 featuredImage: ./images/docs-groups.png
 ---
 
-import CreateGroupType from "./\_snippets/group-type-create.mdx"
-import ChatGroupType from "./\_snippets/group-type-create-chat.mdx"
-import SetGroupProperties from "./\_snippets/set-group-properties.mdx"
+import JSGroups from "../\_snippets/groups.mdx"
```

Comment:
We shouldn't have two pages with the exact same content if it is this long. We can just link to the `product-analytics/group-analytics` page in the getting started sidebar if we don't want another page.

Reply:
Alternatively, the getting started section should have some basic details on set up and usage while the main page has the full context. You can use a part as a snippet in that situation.

Reply:
Fair point. 

> Alternatively, the getting started section should have some basic details on set up and usage while the main page has the full context. You can use a part as a snippet in that situation.

I think I'll go with this solution

Reply:
Another issue with this snippet approach is the page nav in the right sidebar doesn't load. 

<img width="325" alt="Screenshot 2023-05-23 at 10 39 59" src="https://github.com/PostHog/posthog.com/assets/92976667/6c0f6fc6-95b2-4b0a-bd7c-3dcf1e9c1c22">